cmd/goal/tealsign.go

```diff
@@ -46,14 +46,14 @@ func init() {
 	clerkCmd.AddCommand(tealsignCmd)
```

Comment:
It might be useful to say in the help text somewhere which arguments are mutually exclusive.

Reply:
added better help text